app/src/main/res/values/strings.xml

```diff
@@ -1,34 +1,114 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="nav_header_desc">Navigation header</string>
-  <string name="menu_options">Options</string>
-  <string name="menu_my_downloads">My Downloads</string>
-  <string name="menu_help">Help</string>
-  <string name="exploration_activity_title">Exploration Player</string>
-  <string name="help_activity_title">Help</string>
-  <string name="bottom_sheet_options_menu_close">Close</string>
-  <string name="menu_switch_profile">Switch Profile</string>
-  <string name="developer_options">Developer Options</string>
+  <!-- AdministratorControlActivity -->
+  <string name="administrator_controls_title">Administrator Controls</string>
+  <string name="ongoing_topic_list_activity_title">Topics in Progress</string>
   <string name="administrator_controls">Administrator Controls</string>
-  <string name="drawer_open_content_description">Navigation Menu Open</string>
-  <string name="drawer_close_content_description">Navigation Menu Close</string>
+  <!-- AdminAuthActivity -->
+  <string name="admin_auth_toolbar">Administrator Controls</string>
+  <string name="admin_auth_activity_add_profiles_title">Authorise to add profiles</string>
```

Comment:
So these ought to also be in alphabetical order, but I think we also need to update the string names to be clearer.

Technically the second issue is unrelated to #3384, but would you be willing to update the strings while you're here and changing this file? We can chat about naming rules to follow.

Reply:
Yes sure, but can you please explain what to update. It's not clear to me.

Reply:
Thanks! I think the main thing would be updating all of these to be properly "bucketed" within their corresponding activity. Some examples:
- admin_auth_toolbar would become admin_auth_activity_toolbar_title
- admin_auth_activity_add_profiles_title would stay the same
- admin_auth_heading would become admin_auth_activity_heading

Basically, we make sure that:
- The first parts of the string exactly correspond to the activity that they're part of.
- There's no redundancy (e.g. we wouldn't want admin_auth_activity_admin_auth_toolbar.
- The part after the activity name clearly describes what the string is used for (NOT what it contains). Notice how I updated admin_auth_toolbar to include "title" at the end since that's what it is--it isn't a toolbar, it's a toolbar title.

The last point above may be a bit tricky, so I suggest moving any strings that you can't come up with clear names for in a separate section at the bottom with the comment ``<!-- TO BE PLACED -->``. This will be temporary and I can provide suggestions in a follow-up comment.

For any strings that are used in more than one location, please add those to a section at the bottom with the header ``<!-- Multi-screen. -->`` and keep the names as-is; I'll need to think about what to do with those once they're categorized.

Reply:
Okay, that's all do for all strings?

Reply:
Yeah, we've been meaning to do this for a while since it stops the current status quo issue of not having clear string examples to reference for good naming. This may lead to quite a few changes, so I'd understand if you didn't want to tackle it. That being said, properly and fully tidying up the strings file will have a lot of benefit for the team long-term so it would be nice to get done. :)

Reply:
Yey, I will complete it. If stuck anywhere I comment there.